.env

```diff
@@ -0,0 +1,4 @@
+DEVLANDIA_PORT=8000
```

Comment:
I don't think you intended to include this file?

Reply:
all of the previous ones had a .env file with this in it, so I included it, I'm not sure if it actually does anything, and I did not test removing it but I can

Reply:
actually, don't remove it. I discovered that it's necessary for stuff like `jb manage` and `jb run` to work -- or anything else that ends up invoking docker-compose, because otherwise the docker-compose.yml can't be parsed because the variables will be undefined.

Reply:
OOPS sorry I meant to put this comment in response to the other thread about generating .env

I don't think we need to include this .env file inside of the devlandia repo in this PR, though, just keep auto-generating it

Reply:
aright sounds good